### PR TITLE
Update flake8 and fixed lint

### PR DESCRIPTION
**Pull request by kchygoe**
- updated flake8 package
- fixed the lint along with new flake8 rules

## Link to any relevant issues or pull requests.

### CODE REVIEW

File: blueoil/datasets/pascalvoc_base.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def _image_ids(self, data_type=None):
             return all_image_ids
 
         image_ids = [
-            image_id for image_id, gt_boxes in zip(all_image_ids, all_gt_boxes_list) if len(gt_boxes) is not 0
+            image_id for image_id, gt_boxes in zip(all_image_ids, all_gt_boxes_list) if len(gt_boxes) != 0
```
Comment (iizukak) [inline blueoil/datasets/pascalvoc_base.py]: `is not` is not allowed by newer flake8?

Comment (kchygoe) [reply blueoil/datasets/pascalvoc_base.py]: no, errors `F632 use ==/!= to compare str, bytes, and int literals` are appeared

File: blueoil/cmd/output_event.py [hunk 2]
```diff
@@ -97,17 +109,16 @@ def output(tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="m
 
 output csv: {}
 output md: {}
-""".format(output_csv, output_md)
+""".format(
+        output_csv, output_md
+    )
```
Comment (tfujiwar) [inline blueoil/cmd/output_event.py]: You can write this in a single line.

File: blueoil/datasets/pascalvoc_base.py (hunk 1 above)
Comment (iizukak) [reply blueoil/datasets/pascalvoc_base.py]: I see.

File: blueoil/cmd/output_event.py [hunk 3]
```diff
@@ -34,15 +34,21 @@ def _value_step_list(event_accumulator, metrics_key):
...
         return []
 
 
 def _column_name(event_accumulator, metrics_key):
     return "{}:{}".format(os.path.basename(event_accumulator.path), metrics_key)
 
 
-def output(tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="metrics"):
+def output(
+    tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="metrics"
```
[the remark this answers is not in the document]
Comment (kchygoe) [reply blueoil/cmd/output_event.py]: sorry these was formatted by the formatter I mis-configured. thank you!

File: blueoil/datasets/pascalvoc_base.py (hunk 1 above)
Comment (tsawada) [reply blueoil/datasets/pascalvoc_base.py]: `is` compares identity (whether two reference points to the same object). `==` compares values.
there's no point using `is` when comparing int values.

**Pull request merged**